src/test/java/uk/gov/hmcts/reform/civil/handler/tasks/PollingEventEmitterHandlerTest.java

```diff
@@ -130,4 +132,25 @@ private BusinessProcess businessProcessWithCamundaEvent(String camundaEvent) {
             .status(READY)
             .build();
     }
+
+    @Test
+    public void doesNotRepeatEventClaim() {
```

Comment:
This test is not comprehensive.

It tests that you don't call Camunda twice for the same case with the same event, but what about the same case but a different event? That should probably be called (if I'm reading correctly your intentions) but you're not verifying it in the test.

Add to the `@BeforeEach` method a `caseDetails3b`, with the same `id` but a different `businessProcessWithCamundaEvent`. Then add it to the list of cases you are setting up on line 138 and 141.

Reply:
Added in the new test not to affect other tests